spec/features/budgets/investments_spec.rb

```diff
@@ -311,7 +311,6 @@ def investments_order
       select  'Health: More hospitals', from: 'budget_investment_heading_id'
       fill_in 'budget_investment_title', with: 'Build a skyscraper'
       fill_in 'budget_investment_description', with: 'I want to live in a high tower over the clouds'
-      fill_in 'budget_investment_external_url', with: 'http://http://skyscraperpage.com/'
```

Comment:
I can see 3 more `budget_investment_external_url` occurences in the codebase 🤔  in two more tests:

Reply:
Yes, in management panel also there is a new budget investment form with this field ☺️